nmagent/client.go

```diff
@@ -154,6 +154,33 @@ func (c *Client) DeleteNetworkContainer(ctx context.Context, dcr DeleteContainer
 	return nil
 }
 
+// GetHomeAzInfo gets node's home az info from nmagent
+func (c *Client) GetHomeAzInfo(ctx context.Context) (HomeAzInfo, error) {
+	getHomeAzInfoRequest := &GetHomeAzInfoRequest{}
+	var out HomeAzInfo
```

Comment:
can out be renamed to something that better reflects what it is? Is this the style used in other places in this file as well?

Reply:
Ya. in some places.